cloud/scope/powervs_cluster.go

```diff
@@ -1511,8 +1511,6 @@ func (s *PowerVSClusterScope) validateSecurityGroupRule(originalSecurityGroupRul
 	}
 
 	protocol := string(rule.Protocol)
-	portMin := rule.PortRange.MinimumPort
-	portMax := rule.PortRange.MaximumPort
 	icmpCode := rule.ICMPCode
```

Comment:
Please move this to icmp block!

Reply:
It is getting used in tcp block. So I have moved it to that block 
https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/pull/2011/files#diff-a12914f75cce0ea4c617f28f71fcfbd158441b44cb39595fe88f16c69c269b79R1532

Reply:
have addressed it